### PR TITLE
Address PG Feedback

### PR DESCRIPTION
**Pull request by jakiefermsft**

### CODE REVIEW

**Review by mikaelweave — APPROVED** [review]
This looks great!! 👍🏽

File: src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Converter.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            var resultString = this.dataProcessor.Convert(inputContent, this.rootTemplate, templateProvider, traceInfo);
+            var result = new ConverterResult(ProcessStatus.OK, resultString, traceInfo);
+            SaveResult(outputFile, result);
+        }
+
+        public void BatchFiles(List<string> files, string outputFolder)
+        {
+            foreach (var file in files)
+            {
+                Console.WriteLine($"Processing {Path.GetFullPath(file)}");
```
Comment (jasonheld-msft) [inline src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Converter.cs:45]: Intentionally left in?

Comment (jasonheld-msft) [inline src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Converter.cs:45]: Intentionally left in?

**Review by jasonheld-msft — APPROVED** [review]

File: src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Converter.cs (hunk 1 above)
Comment (jakiefermsft) [reply src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Converter.cs:45]: Yep, they are [ported from the original code](https://github.com/jasonheld-msft/FHIR-Converter/blob/main/src/Microsoft.Health.Fhir.Liquid.Converter.Tool/ConverterLogicHandler.cs#L64)

**Pull request merged**